collections/co_3daily_data.yaml

```diff
@@ -0,0 +1,83 @@
+Name: CO 3daily data
+Description: |
+  This online platform uses data from the Copernicus Sentinel-5P satellite and shows the averaged carbon monoxide concentrations across the globe — using a 3-day moving average. Using a 3 day average eliminates most data gaps and improves data quality by reducing random noise. Satellite measurements of concentrations of pollutants like Carbon Monoxide, having a life-time in the atmosphere of about 1 month, can be used to monitor trans-boundary movement of air pollution.
+  Carbon monoxide (CO) is a colorless, odorless, and tasteless gas that is toxic to humans as it can disrupt the transport of oxygen by the blood. In the atmosphere, it is spatially variable and has a life-time of about 1 month.
+  It is produced primarily by the incomplete combustion or oxidation of carbonaceous materials or substances used for fuel in transportation by internal combustion motor vehicles, in industrial heating and processing, and in the heating of homes and buildings. Other sources of CO production are open burning and aircraft emission.
+  The Copernicus Sentinel-5P CO measurements were first filtered according to the recommendation in the Product Readme file (only data with a qa_value > 0.50 was used). Then the measurements are mapped on a fixed latitude-longitude grid of 4096 x 8192 pixels. The grid is turned into an EPSG:4326 geotiff file using the appropriate color scale, which is again turned into an EPSG:3857 tile map.
+AdditionalInfoExternal:
+  Title: Additional info
+  Path: co_3daily_data/README.MD
+Image: co_3daily_data/co.png
+EDCBrowser: ""
+Resolution: 4.8 x 4.8 Km (across x along track)
+GeographicalCoverage: -180.0, -90.0, 180.0, 90.0
+TemporalAvailability: 2018-04-30 - ongoing
+UpdateFrequency: daily
+BandInformation: co
```

Comment:
Please follow the [default structure](https://github.com/eurodatacube/public-collections/blob/main/CONTRIBUTING.md#yaml-file-structure) of `BandInformation`.